tokens/woo.json

```
@@ -0,0 +1,11 @@
+{
+    "ethereum_address": "0x4691937a7508860f876c9c0a2a617e7d9e945d4b",
+    "aurora_address": "0x99ec8f13b2afef5ec49073b9d20df109d25f78c0",
+    "name": "WOO Network",
```

The token name should be the same as the ERC-20 token on Ethereum.

```suggestion
    "name": "Wootrade Network",
```